frontend/src/components/header/Header.tsx

```diff
@@ -317,9 +325,14 @@ export default function Header({
   const onNotificationsClose = () => setAnchorEl(null);
   const getLogo = () => {
     let imageUrl = "/images";
-
-    if (isCustomHub) {
-      return `${imageUrl}/hub_logos/ch_${hubUrl}_logo.svg`;
+    if (!isCustomHub) {
+      if (isLocationHub && hubUrl) {
+        imageUrl += `/hub_logos/ch_${hubUrl}_logo.svg`;
+      } else {
+        imageUrl = loadDefaultLogo(transparentHeader, isMediumScreen);
+      }
+    } else {
+      imageUrl += `/hub_logos/ch_${hubUrl}_logo.svg`;
```

Comment:
It seems like this change just breaks the functionality. If it's a location hub the image url will be imageUrl +  `/hub_logos/ch_${hubUrl}_logo.svg` + `/hub_logos/ch_${hubUrl}_logo.svg`. Because the ending is there twice, it defaults back to the default logo